lms/static/js/views/fields.js

```diff
@@ -20,13 +20,16 @@
             tagName: 'div',
 
             indicators: {
+                'canEdit': '<i class="icon fa fa-pencil message-can-edit"></i>',
```

Comment:
all of these icon fonts should have ```aria-hidden="true"``` attributes on them.  If they appear by themselves, without text, then screen reader only text must be added that indicates the purpose of the icon i.e. ```<span class="sr">Editable</span>```